modules/ROOT/pages/index-self-managed.adoc

```diff
@@ -108,6 +108,14 @@ Runtime Fabric on Self-Managed Kubernetes does not include external log forwardi
 
 For Titanium customers, Runtime Fabric on Self-Managed Kubernetes supports logging using Anypoint Monitoring. See xref:monitoring::logs.adoc[Logs in Anypoint Monitoring] for more information.
 
+=== Monitoring
+
+xref:monitoring::index.adoc[Anypoint Monitoring] provides basic metrics for applications and API gateways deployed to Runtime Fabric. 
```

Comment:
`basic` jumps out at me here. Is there a reason for including it?

Reply:
I took that from existing language in the docs, but I agree it's not needed. I'll cut it